### PR TITLE
Handle all stops error

### PR DESCRIPTION
**Pull request by ywsang**
Issue #91

### CODE REVIEW

File: TCAT/Controllers/AllStopsTableViewController.swift
Comment (myo3) [inline TCAT/Controllers/AllStopsTableViewController.swift]: put comment describing why set `contentOffset` to `.zero`

File: TCAT/Controllers/AllStopsTableViewController.swift
Comment (myo3) [inline TCAT/Controllers/AllStopsTableViewController.swift]: @mattbarker016 think Yana mentioned this to you at the meeting but think it would be good to have a custom icon for empty all stops

File: TCAT/Controllers/AllStopsTableViewController.swift
Comment (myo3) [inline TCAT/Controllers/AllStopsTableViewController.swift]: write clarifying comment why add "#"

File: TCAT/Controllers/AllStopsTableViewController.swift
Comment (myo3) [inline TCAT/Controllers/AllStopsTableViewController.swift]: write clarifying comment on why set "#" to numberBusStops

**Review by myo3 — APPROVED** [review]
Looks good! Had some stylistic comments mainly. And let's wait till we get a custom allstops empty icon from Maya or Matt before merging into master

File: TCAT/Controllers/AllStopsTableViewController.swift
Comment (myo3) [inline TCAT/Controllers/AllStopsTableViewController.swift]: Remove space at end of string

File: TCAT/Controllers/AllStopsTableViewController.swift
Comment (mattbarker016) [reply TCAT/Controllers/AllStopsTableViewController.swift]: Agreed, let's just use this one for now

File: TCAT/Controllers/AllStopsTableViewController.swift
Comment (mattbarker016) [inline TCAT/Controllers/AllStopsTableViewController.swift]: Love this (we use them elsewhere too 🙂), just get ride of that one space at the end!

**Review by mattbarker016 — COMMENTED** [review]
One slight spacing nitpick, good otherwise!

File: TCAT/Controllers/AllStopsTableViewController.swift
Comment (AAAstorga) [reply TCAT/Controllers/AllStopsTableViewController.swift]: you should also remove `()` around the parans in the if statement. just `if !allStops.isEmpty`. This makes it cleaner :)

**Review by AAAstorga — COMMENTED** [review]
Really good! Make sure to run `swiftlint autocorrect` before merging!

File: TCAT/Controllers/AllStopsTableViewController.swift
Comment (ywsang) [reply TCAT/Controllers/AllStopsTableViewController.swift]: I changed the image to the big location pin -- can't remember if we decided to wait for a completely different image to merge or just go ahead with the pin?

**Pull request merged**
